app/assets/v2/images/hackathon/onboarding_v2/avatar.svg

```diff
@@ -0,0 +1 @@
hEisB+jd6CQIVBFAJJaM3odOKiew8x42ELC+sWvfgU4qJ0AryxV35xmtFm/FK5cIJxBxoSWxEPYP6JxWiYQTCCJxvkG4VS4RvQOehEicD+hs2aJf8kYPHxj1c0wIt15d48Pe3hsnyX0BYeiLByCSVyXgEG9PCVxIRCRTXtFp70/9zv2Hhn65foRIpnRAv2QKh+m0D6YXZPyLcGvC+DjGGEang0gKGoikIGE/u93OxK98v2qkjyvtY/sjkjEL5xg4hzG9XneCCl8RyYwP9HqHM0vy+8kgxrTNEck0D9dhzXlUp3vYthdWmcs26RaIZJKG/S6ByFFyTYf0y96YHpG8QaKYPM+QS3O1uID41iEQyVsmLtzofcxPKJ8/fyyhwSJEUuEDcpic+ia6RkQrUu4MiKScixNITmfWo6NeBQkWI5I5PqCQK4frJp8/fWREa44fIJI5cJT10/EXzxrbna2TQE4t5irWQiQeas6J7Jm2qanpJ4FN2A2RLEBRZ9om3sp6fPw5i3ByARPPXQWRzMUzzvz29ahRt/jqgmn3kAmdYwtXf0Mk1WymcjQc/O3rHxoxLKxW8bjhfa0p4635A5EsAVD9kz/98XirhaI6qFUkLU4AkSzOyq25zULZ5rIvaaaNro5IVsC5jc6mEGvbW8EVTLWRTRDJihgllL/8+aetGB3SRVEEsqKh7WaIZHV27iq1nC/VyZAabNAw79HRpzVqyabcXbOmD8gR5YR6btfZ2YUZDtN4J7zCq5+sQLiBak0D280RyfoM3R6KWbSXlzfm8urGPNf0/nM9xEGi5ckvGzIsItkcSO1JrYquzne7bXN+fmX6t4NoYpE43LG5QLhZo9q90ZJsHKmFau8R18W6I9uaXN/cmuvrvnl4eAxwJONajEMrSlqOIHjdThFJOLauZVGnXn/D4dC1LIO7e3M3uF+5hVGLoZuj1AeSMNR6kcISQCRh+Y72rtalEIwWSjSPtpMv0SipD3M/09oUdwrqvYUactarEBCFwxX1HyKJint8MIlGf4UQxjl8S40AbXVqFqE8yRFAJMmZhAKlRgCRpGYRypMcAUSSnEkoUGoEEElqFqE8yRFAJMmZhAKlRgCRpGYRypMcAUSSnEkoUGoEEElqFqE8yRHgivuGTVJMN9FuNanx+el5dATN2ZpNmo4yb/KjpqFoSspsej+zfO/DeMoK01dmaa33G5EsyU8O/WQdu5hzVTi+E4RdvukkEQ1KxKXj9G/v5h5OkyE19aUQVGv35ffu6/K5G5M5IoBIRiimv8j51SoMH5/crF33PZG7DqdLWv1Ld0kWd0qWCUp3LxYCUkukFkiiIk0TgIjlIUGoJXi4f3AzceeFP9P4tvtX0ULNCkjikWBehLNXGu5td82XK312IlGL4ERx/zgSx3LImr+2xFMIqKithKMZyxKOPnOast94kRSiKG52KsKPwvh8LkZgVjgaTJBYdPNX00XTOJGoo6uW4rZ/5z4RxWIiWHYtF57aEFUPvVCSaHSnZKdz0LjwrBEiUWuhhy70rTBmw4Rljc/6qxEoRHN+ceVCsU5737StYJrQymytSGQUtRghH7KwmruwlVpz9wAM+xAMJbUw7c6+E8w2jp5tlUg0CqMwKuajenD59QnIbsUIWtGX6fUOt2a4OXmRFGely8vr0Zj/+mZjD3URKMIy9WU0YqZnhSkkSzklLZLrm759dOjlyo/fSRk8ZTOu//j9+2nyYklSJDrb/Hr6+9w5TThZcwhosEVi0SOX1LKkdg0mOZHQejTH+ZetiUIwDcbojcBlkzqX3d+m1k9qqvypbT1OT88JrzZl3S3cj6KIX2yrUnT0U6hCMiKRQDRsSIKABmtOTs6sP/STgJGESBBIEr6QXCEUVaQglNpFolcU0IIk55/JFEijmwrB6ky1ikRxp6YxkCBQRUCh14+T32rtp9YmElVezSkJAj4CmqRap6/UJhJGsXyuQf4kgcmpLZPLY3yvRSQaC09piC8GaI6xPgG9uLWOVItI1FknQWBZAgq76vCd6CJRK8I9H8u6B+sXBOoYEo4ukiv7CmcSBFYloNYktlCiikQjWvRFVnUPtisI3PYHxdcon1FFgkCi2LTxB5Ef6YQbK0UVSewzQCyIHCc+gZgn3KgiUaedBIFNEHiwz02LlaKJRE80idlExgLIceohMPvO+5CliCaSRzsqQYLApgjEvIwQTSTFU9g3BYn9QEDRSYwUTSSxKhQDGsdIg0Cs6CSaSGJVKA3zUYoYBGJFJ9FEEjOGjGEgjlE/gVjRSRSRxKpM/WajBDEJxIpOoogkVmViGohj1U8gVnQSRSSxYsf6zUYJYhOIEaVEEUmMisQ2DsdLg0CMKCWKSGJUJA2TUYrYBGJEKVFEEit2jG0gjlc/gRhRSnCRxKhE/aaiBHURiBGlBBdJjErUZSCOWz+BGFFKcJHEiBnrNxUlqJNA6GgluEhCV6BO43DsNAiEjlaCiyR0BdIwE6Wok0DoaCW4SGLEjHUaiGPXTyB0tBJUJKELX795KEEKBEJHK0FFErrwKRiIMtRPIHS0ElQkoWPF+s1DCVIhEDJqCSqSkAVPxTiUIw0CIaOWoCKJ+USLNExFKeoiEDJqCSqSul/jVZfBOG58AiF9LZhIQhY6vgk4YuoEHgM+OSWYSEIWOnWDUb74BEKelIOJJOZjKOObhCOmSCCUUMKJpObXCqdoRMoUlkCo6CWYSEIVOCxm9r7NBEJFL8FEEqrp22YjUvawBEL5XBCRhCpsWMTsfdsJhIpegogkVGG33YiUPyyBUCfnICIJFRuGRczem0AghFCCiKQJsKnDdhLY2Xm38YIHEUmnc7DxgrJDCPgItFrvTavV8q22dH4Qkezt7ZruYXvpwrABBNYhcHz8ZZ3NK7eV7C7s38+Va6yYoQIftA++3d0Nvq64CzaDwEIEdt69G3a7nX/Zk/PNQhssudL/AftBaVgY9/AkAAAAAElFTkSuQmCC"/></defs></svg>
```

Comment:
lets avoid base64 images on svgs, when is a bitmap just use png or jpg.
in this case you can even link the avatar from the user from the system if you want.

Reply:
:grimacing: thanks for paying attention to that. Fix incoming

Reply:
Turned out I was even done with the image. The file has been deleted